angular/projects/spark-angular/src/lib/components/sprk-card/sprk-card.stories.ts

```diff
@@ -48,21 +845,20 @@ export const defaultStory = () => ({
         sprk-c-Card__content
         sprk-o-Stack
         sprk-o-Stack--medium">
-        Base Card Content
+        <p class="sprk-TypeBodyTwo">Default Card</p>
```

Comment:
missing `sprkText`

Reply:
@bobwise that is a legacy story, the old code API.